### PR TITLE
Editorial: use proper cross-references for DOM, FileAPI and some of HTML.

### PR DESCRIPTION
**Pull request by Ms2ger**
***
<a href="https://pr-preview.s3.amazonaws.com/Igalia/webdriver/pull/1689.html" title="Last updated on Oct 19, 2022, 12:37 PM UTC (0409e1c)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webdriver/1689/0707ea4...Igalia:0409e1c.html" title="Last updated on Oct 19, 2022, 12:37 PM UTC (0409e1c)">Diff</a>

### CODE REVIEW

**Comment by whimboo** [comment]
Note there is a build failure in CI: `Error: Bad request - pnpm/action-setup@v2.2.2 is not allowed to be used in w3c/webdriver. Actions in this workflow must be: within a repository owned by w3c or created by GitHub.` I assume something has changed to this dependency and is not related to this PR?

**Comment by Ms2ger** [comment]
Not intentionally, but https://github.com/w3c/spec-prod/pull/148 looks highly suspicious. CC @sidvishnoi

**Comment by sidvishnoi** [comment]
Indeed. spec-prod now uses pnpm. Please add `pnpm/action-setup` to actions allow list.

If there's some specific reason to not use action-setup from pnpm org, let me know so I can look into alternate approaches to enable pnpm (maybe with Node.js experimental corepack feature).

**Review by jgraham — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -10189,7 +10191,7 @@ <h2>Screen capture</h2>
   then return <a>error</a> with <a>error code</a> <a>unable to capture screen</a>.
 
  <li><p>Let <var>file</var> be
-  <a>a serialization of the <code>canvas</code> element’s bitmap as a file</a>,
+  [=a serialization of the bitmap as a file|a serialization of the <code>canvas</code> element’s bitmap as a file=],
```
Comment (jgraham) [inline index.html:10194]: Oh, I think this one with `<code>` inside the link text is broken.

**Review by jgraham — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Ms2ger) [reply index.html]: Good catch. Fixing this made me realize that this isn't exported from html anyway, so I restored the link in the index.

**Review by jgraham — APPROVED** [review]

**Pull request merged**